### PR TITLE
8492: Stop logging successful health checks

### PR DESCRIPTION
**Pull request by RuthBerry96**
Story: [AB#8492](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/8492)
Task: [AB#8440](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/8440)

### CODE REVIEW

File: src/NHSD.BuyingCatalogue.API/Infrastructure/Logging/SerilogRequestLoggingOptions.cs [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+using Microsoft.AspNetCore.Http;
+using Serilog.Events;
+
+namespace NHSD.BuyingCatalogue.API.Infrastructure.Logging
+{
+    public static class SerilogRequestLoggingOptions
+    {
+        public const string HealthCheckEndpointDisplayName = "Health checks";
+
+        public static LogEventLevel GetLevel(HttpContext httpContext,double value, Exception exception)
```
Comment (sam-roberts-bjss) [inline src/NHSD.BuyingCatalogue.API/Infrastructure/Logging/SerilogRequestLoggingOptions.cs:11]: Very minor -> parameter spacing

File: src/NHSD.BuyingCatalogue.API/Startup.cs [hunk 2]
```diff
@@ -60,7 +60,6 @@ public void ConfigureServices(IServiceCollection services)
 
             services
                 .AddSingleton<ISettings>(settings)
-                .AddTransient(typeof(IPipelineBehavior<,>), typeof(LoggingBehaviour<,>))
```
Comment (sam-roberts-bjss) [inline src/NHSD.BuyingCatalogue.API/Startup.cs:63]: Any reason why this got removed?

File: tests/NHSD.BuyingCatalogue.API.UnitTests/Logging/SerilogRequestLoggingOptionsTests.cs [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Http.Features;
+using Moq;
+using NHSD.BuyingCatalogue.API.Infrastructure.Logging;
+using NUnit.Framework;
+using Serilog.Events;
+
+namespace NHSD.BuyingCatalogue.API.UnitTests.Logging
+{
+
```
Comment (sam-roberts-bjss) [inline tests/NHSD.BuyingCatalogue.API.UnitTests/Logging/SerilogRequestLoggingOptionsTests.cs:12]: Extra line?

File: src/NHSD.BuyingCatalogue.API/Startup.cs (hunk 2 above)
Comment (RuthBerry96) [reply src/NHSD.BuyingCatalogue.API/Startup.cs:63]: Logging behaviour was removed, as it wasnt being used anymore. There was no need for the transient to be used

**Pull request merged**